abipy/abio/inputs.py

```diff
@@ -1077,35 +1077,110 @@ def make_ddk_inputs(self, tolerance=None):
 
         return ddk_inputs
 
-    def make_dde_inputs(self, tolerance=None):
+    def make_dde_inputs(self, tolerance=None, use_symmetries=True):
         """
         Return inputs for the calculation of the electric field perturbations.
 
         This functions should be called with an input the represents a gs run.
+        Args:
+            tolerance: dict {varname: value} with the tolerance to be used in the DFPT run.
+                Defaults to {"tolwfr": 1.0e-22}.
+
+            use_symmetries: boolean that computes the irreducible components of the perturbation.
+                Default to True. Should be set to False for nonlinear coefficients calculation.
+
+        Return:
+            List of AbinitInput objects for DFPT runs.
         """
         if tolerance is None:
-            tolerance = {"tolvrs": 1.0e-10}
+            tolerance = {"tolvrs": 1.0e-22}
```

Comment:
```
tolerance = {"tolvrs": 1.0e-22}
```

This value is quite small and there are (magnetic) systems in which it's difficult to reach such a value.
I usually prefer `tolwfr = 1.0e-20` when I want accurate wavefunctions (both occupied and empty states)
to be used in DFPT calculations. The main disadvantage of `tolwfr` is that one should use `nbdbuf` to 
exclude some of the highest energy states that tend to oscillate during the iterative solution of the KS equations.

Now I noticed that we are not consistent everywhere with the default tolerances.
There are DFPT calculations in which we use `tolwfr` and other cases in which we use tolwfr

@guido 
Which convergence criterion do you use for phonons. Do you suggest `tolvrs < 1e-10` or `tolwfr < 1e-20`?